db/changes_test.go

```diff
@@ -22,6 +22,87 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// Unit test for CBG-1326
+func TestLogIfChannelsDroppedFromList(t *testing.T) {
```

Comment:
Bit of a nitpick but wondering if this could use testCases? Each test looks the same except the create docs / channels. You could have a parameter that takes "channels list" or something and just create docs based off of that. Reduces a lot of the repetition.
If you think that makes it harder feel free to ignore this one!